<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockDistribution</class>
 <widget class="QWidget" name="BlockDistribution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>127</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="1" column="0">
      <widget class="QToolButton" name="buttonBottom">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="buttonRight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QToolButton" name="buttonMarginVerticalSpace">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="buttonPageVerticalSpace">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="buttonLeft">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="buttonCenter">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="8">
      <widget class="QToolButton" name="buttonSwapLeft">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="buttonPageHorizontalSpace">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QToolButton" name="buttonSwapRight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="buttonTop">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="buttonMiddle">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="buttonMarginHorizontalSpace">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="buttonHorizontalSpace">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QToolButton" name="buttonVerticalSpace">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxSpacing">
         <property name="text">
          <string>Use Spacing:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AdvancedSpinBox" name="numberDistance"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxReverseDistance">
       <property name="text">
        <string>Reverse Distribution</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>advancedspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
